website/docs/index.html.markdown

```diff
@@ -26,6 +26,15 @@ The following vCloud Director versions are supported by this provider:
 * 10.0
 * 10.1
 
+
+## Removed resources
+
+The following resources were removed in *v3.0*:
```

Comment:
That one is broken in the website:
![Uploading image.png…]()

Reply:
fixed